apps/ledger-live-mobile/src/reducers/swap.ts

```diff
@@ -18,7 +19,7 @@ export const INITIAL_STATE: SwapStateType = {
   exchangeRateExpiration: undefined,
 };
 
-export const ratesExpirationThreshold = 30000;
+export const ratesExpirationThreshold = getRatesExpirationThreshold();
```

Comment:
What do you think about remove this extra constant and use directly the value that you are exporting?